source/openapi3/split.yaml

```diff
@@ -5641,5 +5789,97 @@ components:
           amount: 230
           description: Transfer to my other Float account
           matures_at: '2021-05-03T00:00:00Z'
+    ListAllWebhooksResponse:
+      title:  List all Webhooks (response)
+      required:
+        - data
+      type: object
+      properties:
+        data:
+          required: true
+          type: array
+          items:
+            type: object
+      example:
+        data:
+        - id: 31918dce-2dc3-405b-8d3c-fd3901b17e9f
+          url: "http://www.example.com"
+          signature_secret: "top-secret"
+          events:
+          - debit.cleared
+          - credit.cleared
+    GetWebhookDeliveriesResponse:
+      properties:
+        data:
+          required: true
+          type: array
+          items:
+            type: object
+      example:
+        data:
+        - id: 88025fe0-ba80-4cfc-bcc6-c23147ac69a9
+          state: "completed"
+          created_at: "2021-08-22T02:57:45Z"
+          response_status_code: 200
+          event_type: "payout_request.approved"
+          payload_data_summary:
+          - ref: "D.6st94"
+          - ref: "D.6st95"
+          - ref: "D.6st93"
+    GetAWebhookDeliveryResponse:
+      title: Get a WebhookDelivery (response)
+      required:
+        - data
+      type: object
+      properties:
+        data:
+          type: object
+          properties:
+            id:
+              type: string
+              format: uuid
+              description: The Webhook Delivery ID
+            webhook_id:
+              type: string
+              format: uuid
+              description: The Webhook ID
+            state:
+              type: string
+              description: The state of the webhook delivery.
+              enum:
+                - pending
```

Comment:
I believe these are translated into more merchant-friendly terms, for the API.

https://github.com/zeptofs/split/blob/00931172e1fd9ce658112fb378b4879ea6e13dc5/app/decorators/webhook_delivery_decorator.rb#L5-L8